Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackFindingAlgorithm.hpp

```diff
@@ -122,6 +123,14 @@ class TrackFindingAlgorithm final : public IAlgorithm {
     bool stayOnSeed = false;
     /// Compute shared hit information
     bool computeSharedHits = false;
+
+    // Pixel and strip volume ids to be used for maxPixel/StripHoles cuts
+    std::set<Acts::GeometryIdentifier::Value> pixelVolumes;
+    std::set<Acts::GeometryIdentifier::Value> stripVolumes;
```

Comment:
Ah sorry this is not used for the whole ID but just for the volume part. Maybe we can use `std::uint32_t` or `std::size_t`? To make this a bit more explicit

Reply:
isn't is better to have the explicit type `Acts::GeometryIdentifier::Value` the same as the lookup type? If that changes size, it should match, I think.

Reply:
In principle yes. For the volume ID it does not matter much I suppose because they are fairly small numbers. It is just a personal preference on the interface. No need to change it